system/jlib/jsecrets.cpp

```diff
@@ -350,7 +378,7 @@ class CVault
                 StringBuffer tokenText;
                 if (getSecretKeyValue(clientToken, clientSecret, "token"))
                 {
-                    authType = VaultAuthType::token;
+                    authType = VaultAuthType::appRole;
```

Comment:
This looks like some old code sneaked back in.

Reply:
@ghalliday fixed.